src/content/docs/features/window-state.mdx

```diff
@@ -3,12 +3,152 @@ title: Window State
 description: Persist window sizes and positions.
 ---
 
-import Stub from '@components/Stub.astro';
 import PluginLinks from '@components/PluginLinks.astro';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
 
 <PluginLinks plugin="window-state" />
 
-<Stub>
-	Based on
-	https://github.com/tauri-apps/plugins-workspace/tree/v2/plugins/window-state
-</Stub>
+Save window positions and sizes and restore them when the app is reopened.
+
+## Supported Platforms
+
+- Windows
+- Linux
+- macOS
+
+## Setup
+
+_This plugin requires a Rust version of at least **1.75**_
+
+Install the window-state plugin to get started.
+
+<Tabs>
+  <TabItem label="Automatic">
+
+Use your project's package manager to add the dependency:
+
+{' '}
+
+<CommandTabs
+	npm="npm run tauri add window-state"
+	yarn="yarn run tauri add window-state"
+	pnpm="pnpm tauri add window-state"
+	cargo="cargo tauri add window-state"
+/>
+
+  </TabItem>
+  <TabItem label="Manual">
+
+1. Install the Core plugin by adding the following to your `Cargo.toml` file:
+
+```toml title="src-tauri/Cargo.toml"
+[dependencies]
+tauri-plugin-window-state = "2.0.0-beta"
+# alternatively with Git:
+tauri-plugin-window-state = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "v2" }
+```
+
+2. Install the JavaScript Guest bindings using your preferred JavaScript package manager:
+
+  <CommandTabs
+      npm="npm install @tauri-apps/plugin-window-state"
+      yarn="yarn add @tauri-apps/plugin-window-state"
+      pnpm="pnpm add @tauri-apps/plugin-window-state"
+  />
+  </TabItem>
+</Tabs>
+
+## Usage
+
+First you need to register the core plugin with Tauri:
+
+```rust title="src-tauri/src/main.rs" {3}
+fn main() {
+    tauri::Builder::default()
+        .plugin(tauri_plugin_window_state::Builder::default().build())
+        .run(tauri::generate_context!())
+        .expect("error while running tauri application");
+}
+```
```

Comment:
I missed this first time, this step should be in "manual setup" only. Unless the "add" command doesn't add it correctly as in needing additional configuration.